Source/Model/Primitives/SVGFont.swift

```diff
@@ -13,12 +13,33 @@ public class SVGFont: SerializableBlock {
     }
 
     public func toSwiftUI() -> Font {
-        return Font.custom(name, size: size)//.weight(fontWeight)
+        return Font.custom(name, size: size).weight(fontWeight(from: weight))
     }
 
     func serialize(_ serializer: Serializer) {
         serializer.add("name", name, "Serif").add("size", size, 16).add("weight", weight, "normal")
     }
+    
+    private func fontWeight(from: String) -> Font.Weight {
+        // TODO: lighter and bolder are context-based, this is temporary.
+        switch from {
+        case "lighter": return .light
+        case  "normal": return .regular
+        case    "bold": return .bold
+        case  "bolder": return .black
```

Comment:
What's up with these spacings? 😄

Reply:
ocd? hahah I'm just a fan of lined-up code

Reply:
I mean any SwiftFormat/SwiftLint or similar tool would start freaking out here 😄

Reply:
alr I'll use conventions then

Reply:
@msrutek-paylocity I fixed the spacing using swift conventions 🖖